docs/translations.md

```diff
@@ -0,0 +1,146 @@
+---
+category: Develop
+---
+# Translations
```

Comment:
refs #44 I merged the blog post about "Making a plugin multilingual". Before it was named "Internationalization" but only covered translations, therefore renamed it.